docs/docker/nginx.conf

```diff
@@ -0,0 +1,24 @@
+server {
+    listen       80;
+    server_name  localhost;
+
+    gzip on;
+    gzip_http_version 1.1;
```

Comment:
Staat in onze maykin-default project op 1.0 ipv 1.1 - belangrijk?

Reply:
Hmm, goed opgemerkt. Dat kan tegenwoordig wel op 1.1 staan. Dat is ook [default](http://nginx.org/en/docs/http/ngx_http_gzip_module.html#gzip_http_version) en met 1.0 stellen we alleen Windows XP/IE8 gebruikers teleur. Er blijkt ook wel nadelen te zitten aan 1.0: [bron 1](https://serverfault.com/questions/418693/why-nginx-gzip-http-version-defaults-to-http-1-1), [bron 2](https://askubuntu.com/questions/945909/gzip-http-version-1-1-in-nginx-when-using-http-2)